test/fixtures/item_orders.yml

```diff
@@ -1,13 +1,13 @@
 # Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
 
-one:
-  item_id: 1
-  order_id: 1
-  quantity: 1
-  line_item_cost: 1
-
-two:
-  item_id: 1
-  order_id: 1
-  quantity: 1
-  line_item_cost: 1
+#one:
+#  item_id: 1
+#  order_id: 1
+#  quantity: 1
+#  line_item_cost: 1
+#
+#two:
+#  item_id: 1
+#  order_id: 1
+#  quantity: 1
+#  line_item_cost: 1
```

Comment:
I made it match the other fixture files we found. We can delete it, but I chose to match convention.